src/message-handler/include/message-handler/MessageHandlerContainer.h

```diff
@@ -36,6 +36,13 @@ namespace MessageHandlerContainer {
      *@brief get the remote message queue
      *@return A reference to the remote message queue */
     ThreadSafeQueue<MessageDTO>& getRemoteMsgQueue();
+
+    /**
+     *
+     *@brief get the network message queue
+     *@return A reference to the network message
+     */
+    ThreadSafeQueue<MessageDTO>& getNetworkMessageQueue();
```

Comment:
No need for this queue, the RemoteMsgQueue was for the ESP, putting it on the Uart stream was purely temprorary for the old demo

Reply:
I expect it will break the demo we have working now though?

Reply:
Yes this will break the demo though since no more data will be read from the uart. But the uart has bugs anyway in it I think. Though once your code is working in simulation at the end of the sprint, we should be able to send a message via Phone->HM->ESP->ESP->HM->Robot.